common-pipeline/operators/uihints-v3-schema.json

```diff
@@ -652,6 +659,54 @@
           }
         }
       ]
+    },
+    "button_definition": {
+      "description": "Button to be displayed after the table's search bar",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Button identifier used to idenify the button in the callback function",
+          "type": "string"
+        },
+        "label": {
+          "description": "Button label to display inplace of an icon",
+          "type": "object",
+          "$ref": "#/definitions/resource_definition"
+        },
+        "icon": {
+          "description": "URL to button icon to display",
+          "type": "string"
+        },
+        "carbonIcon": {
+          "description": "Name of the Carbon icon to display",
+          "type": "string"
+        },
+        "tooltip": {
+          "description": "Tooltip text to display when the button is hovered",
+          "type": "object",
+          "$ref": "#/definitions/resource_definition"
+        }
+      },
+      "anyOf": [
+        {
```

Comment:
Changes look good.  Once question is here.  Wouldn't either `icon or carbonIcon` always be required?

Reply:
If no icon is specified, then the button will just be a label. There are instances where the app does not want any icons